include/boost/url/pct_encoding.hpp

```diff
@@ -466,9 +526,19 @@ pct_encode_bytes(
     characters that are not in the specified
     <em>CharSet</em>.
     The output is written to the destination,
-    which will be truncated if there is
+    and will be truncated if there is
     insufficient space.
 
+    @par Example
+    @code
+    auto dest = new char[MAX_LENGTH];
+    auto encoded = pct_encode(dest, dest + MAX_LENGTH,
+        "My stuff");
+
+    assert(encoded == 10);
+    assert(strncmp("My%20stuff", dest, encoded) == 0);
+    @endcode
+
```

Comment:
Same notes regarding examples should be applied to all these changes

Reply:
Sure thing.